CHANGELOG.md

```diff
@@ -24,6 +24,9 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Fix `propTypes` warning in `ListItem` @layershifter ([#1266](https://github.com/stardust-ui/react/pull/1266))
 - Expand css shorthands for correct merging of the styles @mnajdova ([#869](https://github.com/stardust-ui/react/pull/869))
 
+### Features
+- Export call-dialpad icon in Teams theme @assamad ([#1271](https://github.com/stardust-ui/react/pull/1271))
```

Comment:
please, enclose `call-dialpad` into backticks (`), the same way it is done for similar changelog entries